client/debian/control

```diff
@@ -4,24 +4,53 @@ Priority: optional
-Vcs-Git: https://github.com/canonical/hardware-api.git [client/hwlib/src]
-Vcs-Browser: https://github.com/canonical/hardware-api/tree/main/client/hwlib/src
+Vcs-Git: https://github.com/canonical/hardware-api.git [client]
+Vcs-Browser: https://github.com/canonical/hardware-api/tree/main/client
 Homepage: https://github.com/canonical/hardware-api
 X-Cargo-Crate: hwlib
 Rules-Requires-Root: no
 
 Package: librust-hwlib-dev
-Architecture: any
-Multi-Arch: same
-Depends: libssl-dev, pkgconf, ${misc:Depends}, ${shlibs:Depends}
+Architecture: amd64
```

Comment:
This looks wrong. Why would you constrain the architecture?

Reply:
This is the only architecture we fully support now, since we primarily use SMBios for collecting hardware data, which works only on amd64 architectures
I've checked that `dmidecode` tool, which uses SMBios as well, also specifies architecture explicitly in its `debian/control` file

Reply:
Ack.